ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/customercenter/viewmodel/CustomerCenterViewModel.kt

```diff
@@ -435,6 +436,16 @@ internal class CustomerCenterViewModelImpl(
         promotionalOffer: CustomerCenterConfigData.HelpPath.PathDetail.PromotionalOffer,
         originalPath: CustomerCenterConfigData.HelpPath,
     ): Boolean {
+        if (!promotionalOffer.eligible) {
+            Log.d(
```

Comment:
Hmm I wonder if this should be a verbose log, since this would be a common case after all... NABD though

Reply:
I went through as debug in iOS, but I can revisit if it's a problem in the future. I thought about debug because our [guidelines](https://docs.revenuecat.com/docs/debugging) suggest `.debug` as a log level for debugging 🤓